### PR TITLE
Bug 1209863 - [Stingray] Home screen card icon image didn't show, r=r…

### PR DESCRIPTION
**Pull request by danhuang1202**
…exboy
- update serialize() in shared/js/smart-screen/cards/deck.js and application.js

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=291d45092284bf1819d7b4745b96c1e2894ca09a

**Comment by rexboy7** [comment]
Let's change comment to : `Bug 1209863 - [Stingray] Fix: home screen card icon image didn't show`

File: shared/js/smart-screen/cards/deck.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
       cardInstance = new Deck({
         name: cardEntry.name,
         nativeApp: cardEntry.manifestURL &&
-          installedApps[cardEntry.manifestURL],
+        installedApps[cardEntry.manifestURL],
```
Comment (rexboy7) [inline shared/js/smart-screen/cards/deck.js]: I think it's better to keep the indent.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=28853f2c27e5de52ccf665e237ba4e407d4a7cfb

**Pull request merged**